app/http.php

```diff
@@ -141,8 +144,61 @@
             }
 
             $dbForConsole->createCollection($key, $attributes, $indexes);
+        }
 
+        if($dbForConsole->getDocument('buckets', 'default')->isEmpty()) {
+            Console::success('[Setup] - Creating default bucket...');
+            $dbForConsole->createDocument('buckets', new Document([
+                '$id' => 'default',
+                '$collection' => 'buckets',
+                'dateCreated' => \time(),
+                'dateUpdated' => \time(),
+                'name' => 'Default',
+                'permission' => 'file',
+                'maximumFileSize' => (int) App::getEnv('_APP_STORAGE_LIMIT', 0), // 10MB
+                'allowedFileExtensions' => [],
+                'enabled' => true,
+                'encryption' => true,
+                'antivirus' => true,
+                '$read' => ['role:all'],
+                '$write' => ['role:all'],
```

Comment:
Are we sure about these permissions ?

Reply:
yes, as permission is set to file level, this shouldn't matter

Reply:
okay 👍